Finished preprocessing the CelebA dataset...
torch.cuda.is_available(): True
gpu_id: 1
Traceback (most recent call last):
  File "/home/yerinyoon/code/anonymousNet/starGAN/stargan.py", line 282, in <module>
    main(config)
  File "/home/yerinyoon/code/anonymousNet/starGAN/stargan.py", line 95, in main
    solver = Solver(celeba_loader, rafd_loader, config, wandb, c_dim)
  File "/home/yerinyoon/code/anonymousNet/starGAN/model.py", line 169, in __init__
    torch.cuda.set_device(self.device)
  File "/home/yerinyoon/anaconda3/envs/pytorch/lib/python3.10/site-packages/torch/cuda/__init__.py", line 326, in set_device
    torch._C._cuda_setDevice(device)
RuntimeError: CUDA error: invalid device ordinal